persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/UserDAO.kt

```diff
@@ -32,13 +32,20 @@ data class UserEntity(
     // later, when API start supporting it, it should be added into API model too
     val availabilityStatus: UserAvailabilityStatusEntity,
 
-    val userTypEntity: UserTypeEntity,
+    val userTypeEntity: UserTypeEntity,
 )
 
 enum class UserTypeEntity {
-    INTERNAL,
 
-    // TODO(user-metadata): for now External will not be implemented
+    /**Team member with owner permissions */
+    OWNER,
+
+    /**Team member with admin permissions */
+    ADMIN,
+
+    /**Team member */
+    INTERNAL, // TODO Kubaz change to member
```

Comment:
I am not sure about making it `team member`.

Because if I am not a team member (just a plain Wire user), and I have another contact like that, they shouldn't have any "membership badge".

Renaming this to `standard` or similar would probably cover better the cases.

Reply:
Good point will rename it in next PR 💪